src/components/SearchBar.Js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
```suggestion
import PropTypes from 'prop-types';

```
Try to keep whitespace consistent across files.